styles/responsive.scss

```diff
@@ -1,3 +1,34 @@
+//remark-highlight optimisation
+
+@media (max-width: 767px) {
+  .remark-highlight {
+    max-width: 93vw;
+  }
+}
+@media screen and (max-width: 845px) and (min-width: 767px) {
+  .remark-highlight {
+    max-width: 60vw;
+  }
+}
+
+@media screen and (max-width: 1019px) and (min-width: 845px) {
+  .remark-highlight {
+    max-width: 58vw;
+  }
+}
+
+@media screen and (max-width: 1020px) and (min-width: 1019px) {
+  .remark-highlight {
+    max-width: 48vw;
+  }
+}
+
+@media screen and (max-width: 1180px) and (min-width: 1021px) {
+  .remark-highlight {
+    max-width: 59vw;
+  }
+}
+
```

Comment:
Um, feels very hacky as for me

Reply:
What's wrong?

Reply:
@alexander-vashchuk @1alexvash 
What's wrong?

Reply:
Feels hacky, we don't have those breakpoints in the project

Reply:
Maybe you should try calc property to limit the width